### PR TITLE
Webview bridge interface

### PR DESCRIPTION
**Pull request by jpyo0613**
https://www.notion.so/flarelane/JavascriptInterface-3ab6ac6e47b34416bf00fd9efe7dff08?pvs=4

### CODE REVIEW

File: FlareLane/Classes/FlareLaneJavascriptInterface.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    private func trackEvent(body: [String: Any]) {
+        if let type = body["type"] as? String {
+            if let data = body["data"] as? [String: Any] {
+                Logger.verbose("trackEvent() type=\(type), data=\(data)")
+                FlareLane.trackEvent(type, data: body["data"] as? [String: Any])
+            } else {
+                Logger.verbose("trackEvent() type=\(type), data=nil")
+            }
+        }
+    }
```
Comment (junyeongCHOI) [inline FlareLane/Classes/FlareLaneJavascriptInterface.swift:54]: data가 nil인 경우에도 trackEvent가 불려야 할 것 같아요

File: FlareLane/Classes/FlareLaneJavascriptInterface.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        case "trackEvent":
+            trackEvent(body: body)
+        default:
+            break
+        }
+    }
+    
+    private func setUserId(body: [String: Any]) {
+        let userId = body["userId"] as? String
+        Logger.verbose("setUserId() userId=\(userId ?? "nil")")
```
Comment (minhyeok4dev) [inline FlareLane/Classes/FlareLaneJavascriptInterface.swift:34]: Delete duplicated logs

File: FlareLane/Classes/FlareLaneJavascriptInterface.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            setTags(body: body)
+        case "trackEvent":
+            trackEvent(body: body)
+        default:
+            break
+        }
+    }
+    
+    private func setUserId(body: [String: Any]) {
+        let userId = body["userId"] as? String
```
Comment (minhyeok4dev) [inline FlareLane/Classes/FlareLaneJavascriptInterface.swift:33]: How to separate the userId is null or exists?

File: Example/FlareLane/Info.plist [hunk 4]
```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleVersion</key>
 	<string>1</string>
 	<key>FlareLaneSwizzlingEnabled</key>
-	<false/>
+	<true/>
```
Comment (minhyeok4dev) [inline Example/FlareLane/Info.plist:24]: Set **false** to test along with other platforms like Firebase.

**Review by minhyeok4dev — CHANGES_REQUESTED** [review]

File: Example/FlareLane/Info.plist (hunk 4 above)
Comment (jpyo0613) [reply Example/FlareLane/Info.plist]: Applied successfully

File: FlareLane/Classes/FlareLaneJavascriptInterface.swift (hunk 3 above)
Comment (jpyo0613) [reply FlareLane/Classes/FlareLaneJavascriptInterface.swift]: Added fallback logic for cases where the userId key is missing

**Review by minhyeok4dev — APPROVED** [review]

**Pull request merged**